src/game.rs

```diff
@@ -408,15 +410,15 @@ fn process_game(
     // Animation to re-center the screen around the player when they
     // get too close to an edge.
     state.pos_timer.update(dt);
-    if !state.pos_timer.finished() {
+    if state.pos_timer.finished() {
+        state.screen_position_in_world = state.new_screen_pos;
+        state.offset_px = Point::zero();
+    } else {
```

Comment:
Yeah, I generally order `if` statements this way, this one must have slipped. +1